rasa/core/nlg/callback.py

```diff
@@ -31,13 +31,15 @@ def nlg_request_format(
     utter_action: Text,
     tracker: DialogueStateTracker,
     output_channel: Text,
+    message: Text,
```

Comment:
We can't modify the signature of a public function in a minor release, it's preferable to use `kwargs` instead to retrieve the `message`.